docs: Create Github marketplace documentation for Flank action (#1701)
Flank/flank: docs/flank-github-action/store_documentation.md

## Action

> This Github action allow running Flank using your Github workflows.

Reviewer: ```suggestion
This Github action allows for running Flank as a Github workflow.
```

| `version`                  | Flank version to run. Minimal supported version is `v21.03.1`. Leaving it blank will fallback to latest version.                                                                                                                                          | `false`  | latest available |
| `service_account`          | Service account to authenticate with. Could be path to file, link to file or file content itself. More information about creating a service account could be found at [documentation](https://flank.github.io/flank/#authenticate-with-a-service-account) | `true`   |                  |
| `platform`                 | Platform to run. Could be `ios` or `android`                                                                                                                                                                                                              | `true`   |                  |

> | `flank_configuration_file` | Flank configuration file. More info how it should look like is in [documentation](https://flank.github.io/flank/#flank-configuration)                                                                                                                     | `true`   |                  |

Reviewer: ```suggestion
| `flank_configuration_file` | Flank configuration file. More information on how it should look like is in [documentation](https://flank.github.io/flank/#flank-configuration)                                                                                                                     | `true`   |                  |
```

## Action

This Github action allow running Flank using your Github workflows.

> Flank is a [massively parallel Android and iOS test runner](https://docs.google.com/presentation/d/1goan9cXpimSJsS3L60WjljnFA_seUyaWb2e-bezm084/edit#slide=id.p1) for [Firebase Test Lab](https://firebase.google.com/docs/test-lab/). 
> 
> Flank is YAML compatible with [the gcloud CLI](https://cloud.google.com/sdk/gcloud/reference/alpha/firebase/test). Flank provides extra features to accelerate velocity and increase quality.
> 

Reviewer 2: ```suggestion
```
Those 2 lines are redundant, could be read from Flank doc.
